datasets/flwr_datasets/merge_splitter.py

```diff
@@ -0,0 +1,75 @@
+"""Resplitter class for Flower Datasets."""
+import collections
+from typing import Dict, List, Tuple
+
+import datasets
+from datasets import Dataset, DatasetDict
+
+
+class MergeSplitter:
+    """Create a new dataset splits according to the `resplit_strategy`.
+
+    The dataset comes with some predefined splits e.g. "train", "valid" and "test". This
+    class allows you to create a new dataset with splits created according to your needs
+    specified in `resplit_strategy`.
+
+    Parameters
+    ----------
+    resplit_strategy: ResplitStrategy
+        Dictionary with keys - tuples of the current split names to values - the desired
+        split names
+    """
+
+    def __init__(
+        self,
+        resplit_strategy: Dict[Tuple[str, ...], str],
```

Comment:
Should we call this `merge_config` or `merge_strategy` or something similar? It's perhaps best to avoid `strategy` since this term is already used for a different concept in Flower.

Reply:
`merge_config` sounds good